src/yarpmotorgui-qt/mainwindow.cpp

```diff
@@ -103,7 +103,8 @@ MainWindow::MainWindow(QWidget *parent) :
     partToolBar->setAllowedAreas(Qt::AllToolBarAreas);
 
     QFontMetrics metrics(f);
-    int w = metrics.width(globalLabel);
+    // int w = metrics.width(globalLabel);
+    metrics.width(globalLabel);
```

Comment:
it seems so but it may still have side-effects, I wanted to have minimal impact...